EasyIntern-Backend/Controllers/AccountController.cs

```diff
@@ -168,6 +167,11 @@ public async Task<IActionResult> Update([FromBody] Dictionary<string, string> mo
 
         foreach (KeyValuePair<string, string> pair in model)
         {
+            if (!await _context.ProfilesettingsOptions.AnyAsync(e => e.Key == pair.Key))
```

Comment:
So, the idea is to limit keys in settings to an approved list of keys, however I cannot find where you defined this list of approved keys